readme.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs');
```

Comment:
I'd prefer to write everything except `gulpfile.js` in TypeScript. We could have a folder named `readme` inside the `src` folder, and this would be `index.ts` and the other one `rules.ts`.